lib/bk/commands/artifacts.rb

```diff
@@ -103,26 +104,23 @@ def call(args: {}, url_or_slug: nil, **options)
           has_next_page = build.jobs.page_info.has_next_page
 
           jobs = build.jobs.edges.map(&:node)
-          Parallel.each(jobs) do |job|
+          bar = ProgressBar.create(total: jobs.count, throttle_rate: 1, format: '%a %t [%c/%C BK jobs]: %B %j%%, %E')
+
+          Parallel.each(jobs, in_threads: 8) do |job|
+            bar.increment
             next unless job.respond_to?(:exit_status)
 
             artifacts = job.artifacts.edges.map(&:node).select { |artifact| glob_matches?(glob, artifact.path) }
             next unless artifacts.any?
 
             color = job_colors[job.exit_status]
             header = color.call(job.label)
-            if job.parallel_group_index && job.parallel_group_total
```

Comment:
sorry I meant to put this comment on the line removing the logged output, not the conditional. 

The intention here is that regardless of parallel or not, we should show output on a job by job basis, rather than job “group” basis.

Reply:
Hm, I'm not sure I agree with that. When not downloading, it would be a lot more user readable if you are looking at the artifacts as a whole without any globs, as if you were trying to figure out what glob to use.

I'm open to trying it though, as y'all are using this subcommand more than I am so far.

Reply:
I'm not suggesting removing the output of artifacts in general, just the information about how parallel a step is running.

i.e. I don't understand the value of the number in the second example:
new
```
rspec: path/to/file.rb
```

current
```
rspec: (1/360) path/to/file.rb
```

Reply:
It mirrors how Buildkite shows labels of builds when there's parallelization. I tried to take inspiration from the web interface when possible.

We don't show any other info to distinguish parallel steps, so we show the parallel our of however many.

Note: there can be multiple of the same index, like when you retry a step.

Reply:
Okay, I added the notation so we know if a job was run in parallel – let me know if you have any other feedback or you're good with merging now!